packages/features/ee/impersonation/lib/ImpersonationProvider.ts

```diff
@@ -103,7 +105,11 @@ const ImpersonationProvider = CredentialsProvider({
       if (impersonatedUser.disableImpersonation) {
         throw new Error("This user has disabled Impersonation.");
       }
-      return auditAndReturnNextUser(impersonatedUser, session?.user.id as number);
+      return auditAndReturnNextUser(
+        impersonatedUser,
+        session?.user.id as number,
+        impersonatedUser.teams.length > 0 // If the user has any teams, they belong to an active team and we can set the hasActiveTeam ctx to true
```

Comment:
NIT - ideally this should be checking for active subscriptions - however, this is only used in an admin setting